### ISSUE TITLE
LinkElement.getCaption() fails with Unable to locate element: {"method":"xpath","selector":".//*[@aria-labelledby]"}

### ISSUE THREAD

**Issue by vaadin-bot** [vaadin/testbench#781]
**Originally by _mvysny2_**

___
Using the following code instead works correctly:
```
((LinkElement) element).findElement(By.xpath(".//a")).getText()
```
Perhaps the getCaption() method should be overridden in LinkElement.

___
 Imported from https://dev.vaadin.com/ issue #18232
